toolkits/springer/packages/springer-listing/scss/50-components/_listing.scss

```diff
@@ -95,3 +94,12 @@
 	margin-right: $listing--view-options-icon-margin-right;
 	fill: currentColor;
 }
+
+@supports (-webkit-line-clamp: 3) {
```

Comment:
> "truncation is not a content strategy" 

…but progressive enhancement ftw!

Reply:
I've some thoughts on this which I'll probably mull over this evening. 

I don't think SN has any 'playbook' on custom properties (🔔 potential for an open space conversation) and there's a few ways something like this use case for `line-clamp` could be approached.

I lean to an 'off by default' but that opens up that question of where the custom property could or should be set. Yet, in this same train of thought I'm thinking having it 'on' would make things a little cleaner as I'm sure there are times that the content that's going to be displayed here could be massive.

LGTM … for now.